test/cluster/test_alternator.py

```
@@ -2084,4 +2086,44 @@ async def test_alternator_mtls_and_plain_http(manager: ScyllaClusterManager, tmp
         alternator_bad.meta.client.list_tables()
 
 
+# The Alternator S3 export feature stores its metadata in tables under the
+# replicated system_distributed keyspace, created automatically when Alternator
+# starts. Verify that these tables are created on every node of a multi-node
+# cluster and each node sees other nodes' data. The test will write one row per node
+# into each table and run SELECT from every node to verify that all rows are visible.
+# Writes and reads use QUORUM so that, on the RF=3 system_distributed keyspace, every read is
+# guaranteed to observe every write.
+async def test_export_to_s3_system_tables_readable_on_all_nodes(manager: ManagerClient):
```

Ruff (0.16.3)

[error] Undefined name `ManagerClient`

(F821)